src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
This is just very confusing and inconsistent. We either rename the component `Weblink` (bad idea), or we should stick to `symfony/web-link`.